.github/workflows/make-test.yml

```
@@ -83,23 +83,23 @@ jobs:
             cxxflags: -Werror
             cmakeflags: -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang
             ctest_args: '-E ""'
-            ldpath: /usr/local/lib64/
+            ldpath: /usr/local/lib64 
           # - distro: 'CentOS 8.4'
           #   containerid: 'gnuradio/ci:centos-8.4-3.10'
           #   cxxflags: ''
           #   ctest_args: '-E ""'
           #   ldpath: /usr/local/lib64/
-          - distro: 'Debian 11'
-            containerid: 'gnuradio/ci:debian-11-3.10'
+          - distro: 'Debian 12'
+            containerid: 'gnuradio/ci:debian-12-3.10'
             cxxflags: -Werror
             ctest_args: '-E "qa_polar_..coder_(sc_)?systematic"'
```

It should no longer be necessary to skip these tests, since Debian's broken libvolk was fixed.

```suggestion
            ctest_args: '-E ""'
```